bi-att-flow-dev/basic/model_nqa.py

```diff
@@ -73,6 +73,9 @@ def __init__(self, config, scope, rep=True):
         self.var_list = None
         self.na_prob = None
         self.decoder_logits_train = None
+        self.translations = None
+        self.tgt_sos_id = 2
+        self.tgt_eos_id = 3
```

Comment:
@dhruv100691 : SOS id and EOS id should be id's that are not already assigned to any other word in our vocab. Using 2 and 3 seem a bit suspicious. Is this okay?

Reply:
haan..kyunki rest of the words ka id 4 se start karte hain specifically...pehle 2 ids NULL or UNK hai..baaki do yeh

Reply:
Oh okay. Cool cool.